lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSignupCard.js

```diff
@@ -30,9 +32,16 @@ class CampaignSignupCard extends React.Component {
         isLoading: false,
         isSignedUp: true
       });
+
+      sendEvent('onboarding-v2', 'signup', this.constructor.name);
     });
   }
 
+  preRefresh() {
```

Comment:
I feel like the controller should remain agnostic to implementation now, so Id rather leave the data tracking here